src/shared/types/api/http-client.ts

```
@@ -38,17 +30,32 @@ axiosClient.interceptors.response.use(
     }
     return response;
   },
-  (error: AxiosError<ApiErrorResponse>) => {
+  async (error: AxiosError<ApiErrorResponse>) => {
     const errorInfo = {
       status: error.response?.status || 'NETWORK_ERROR',
       message: error.response?.data?.message || error.message,
     };
 
+    const originalRequest = error.config as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)